src/sentry/static/sentry/app/views/settings/organization/auditLog/auditLogList.jsx

```diff
@@ -65,33 +64,27 @@ class AuditLogList extends React.Component {
     let {pageLinks, entries, eventType, eventTypes, onEventSelect} = this.props;
     let hasEntries = entries && entries.length > 0;
 
+    let action = (
```

Comment:
this seems unrelated...just confirming you meant to commit this?

Reply:
Yep! It is related in that there was an unnecessary blurb of text next to this control, so I removed it and made the control a page header action instead.